test/e2e/cheerio-impit-ts/actor/package.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "test-cheerio-impit-ts",
+    "version": "0.0.1",
+    "description": "Cheerio Crawler Test - Impit HTTP client",
+    "dependencies": {
+        "apify": "next",
+        "@apify/storage-local": "^2.1.3",
+        "@crawlee/basic": "file:./packages/basic-crawler",
```

Comment:
Huh. Does this install the package from the current git worktree?

Reply:
yes, that is how the e2e tests work here (outside of the PLATFORM type which copies stuff [here](https://github.com/apify/crawlee/blob/master/test/e2e/tools.mjs#L273))

Reply:
Got it, thanks.